sympy/concrete/tests/test_sums_products.py

```diff
@@ -1045,7 +1045,7 @@ def test_is_convergent():
     assert Sum(eq, (x, 1, oo)).is_convergent() is S.true
     assert Sum(eq, (x, 1, 2)).is_convergent() is S.true
     assert Sum(1/(x**3), (x, 1, oo)).is_convergent() is S.true
-    assert Sum(1/(x**(S(1)/2)), (x, 1, oo)).is_convergent() is S.false
+    assert Sum(1/(x**(S.Half)), (x, 1, oo)).is_convergent() is S.false
```

Comment:
`(S.Half)`->`S.Half` globally

Reply:
Except that it may be an argument to a function. I think I have fixed most though.